docs/006-just-in-time-channels.adoc

```diff
@@ -0,0 +1,59 @@
+= ADR 006 - Just-in-time channels
+Philipp Hoenisch <philipp@10101.finance>
+3.0, July 29, 2022: AsciiDoc article template
+:toc:
+:icons: font
+:attributes: 2023-02-13
+
+Status: Draft
+
+This document contains a description about the protocol we use for channel-less payments.
```

Comment:
:upside_down_face: Channel-less was a funny name at the time, but it is a bit misleading for anyone reading who doesn't already know what this is.